src/Controller/MemberController.php

```diff
@@ -64,17 +67,24 @@ public function show(Member $member): Response
      */
     public function edit(Request $request, Member $member): Response
     {
+        $commandId = 0;
```

Comment:
pourquoi tu initialises à 0 ? je pense que c'est inutile puisque tu fait un $command->getId() juste en dessous.

Reply:
c’était pour que ça passe avec les verifs stan ou un autre. Il me demandait de initialiser mais ça ne me plait pas non plu.

Reply:
non je ne pense pas que ce soit le pb. Enlève le et recommence
tu devrais juste avoir 

            return $this->redirectToRoute('member_index', ['command' => $command->getId()]);